### DISCUSSION TITLE
Add the ability to add className to Table

### DISCUSSION THREAD

**Discussion by iakovalgaev** [table-library/react-table-library#34] [Ideas]
`<Body>` component I can replace `<div className='wrapper'>`. But also I want to have possible add styles on `<Table>` without additional wrapper.

**Comment by rwieruch** [comment]
Regarding the Body component: At the moment the Body component is more of a placeholder for the future, if I want to introduce features in there. However, it can be left out at this time.

----

If you want to give the Table component extra style, you need to use `const theme = useTheme(THEME)`:

```
const THEME = {
  Table: `
    background-color: pink;
  `,
};

const theme = useTheme(THEME);

return <Table theme={theme} ... >;
```

If you want to give the Table a CSS class and style it, do it this way:

```
const THEME = {
  Table: `
    &.my-table {
     background-color: orange;
    }
  `,
};

const theme = useTheme(THEME);

return <Table theme={theme} className="my-table" ... >;
```
